tpu-client-next/src/connection_workers_scheduler.rs

```diff
@@ -39,6 +39,15 @@ pub enum ConnectionWorkersSchedulerError {
     LeaderReceiverDropped,
 }
 
+/// This enum defines to how many discovered leaders we will send transactions.
+pub enum LeadersFanout {
+    /// Send transactions to all the leaders discovered by the `next_leaders`
+    /// call.
+    All,
+    /// Send transactions to the first selected number of leaders.
+    Selected(usize),
```

Comment:
I'd call this `Next(usize)` so it's like `LeadersFanout::Next(4)` which sounds
better to me.

The doc should also clarify that the value is clamped by the number of leaders
returned by next_leaders()